README.md

```diff
@@ -28,12 +28,12 @@ So if you want to get all the examples, make sure to clone the repo recursively
-|----------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------|---------------------------------------|
-| singlecam                                                                                                            | Generates spatial motion data using a single video as input, optimised for quality.           | 25-30s for 10s, 60fps, HD             |
-| singlecam-turbo <br/>(coming soon)                                                                                   | Generates spatial motion data using a single video as input, optimised for speed and quality. | 10s for 10s, 60fps, HD                |
-| multicam <br/>(coming soon)                                                                                          | Generates spatial motion data using multiple videos as input, optimised for quality.          | 2.5mins for 10s, 60fps, HD, 4 cameras |
-| multicam-rt <br/>([available now, request access here](https://share-eu1.hsforms.com/1J1WzWmHUT_aXIlmv7-b3xwfk5ge))  | Generates spatial motion data using multiple videos as input, optimised for real-time speed.  | <120ms latency                        |
+| Model                                                                                                               | Description                                                                                   | Inference time                        |
+|---------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------|---------------------------------------|
+| singlecam                                                                                                           | Generates spatial motion data using a single video as input, optimised for quality.           | 25-30s for 10s, 60fps, HD             |
+| singlecam-turbo <br/>(coming soon)                                                                                  | Generates spatial motion data using a single video as input, optimised for speed and quality. | 10s for 10s, 60fps, HD                |
+| multicam                                                                                          | Generates spatial motion data using multiple videos as input, optimised for quality.          | 2.5mins for 10s, 60fps, HD, 4 cameras |
+| multicam-rt <br/>([available now, request access here](https://share-eu1.hsforms.com/1J1WzWmHUT_aXIlmv7-b3xwfk5ge)) | Generates spatial motion data using multiple videos as input, optimised for real-time speed.  | <120ms latency                        |
```

Comment:
I don't think we need the request access form? Double check with Ross/Jack

Reply:
That's for real time I think? that's why I've left that as it is

Reply:
We should still flag it with Ross - because if it's the same form and he's changing where the form goes to then he needs to know. (even though you haven't changed it)

Reply:
I've flagged up with Jack/Ross - Jack has no idea whether we use this link or not - still yet to receive a response from Ross. Should we merge this pull request and update it later if required (since we're not actually changing the URL right now)?